### ISSUE TITLE
Dezoomify topo map

### ISSUE THREAD

**Issue by srdjantodorov93** [lovasoa/dezoomify#380]
I try everything to dezoomify large topo map from this site, but unsuccessful.
https://geoportal.asig.gov.al/map/?fc_name=25k_krgjsh

Map is important to studying geography.

Thanks in advance!

**Comment by lovasoa** [comment]
The map is very large, I advise you to use [dezoomidy-rs]() instead of dezoomify : 

```
dezoomify-rs --retries 0 "https://geoportal.asig.gov.al/service/wmts?layer=orthophoto_2007%3AOrthoImagery&style=default&tilematrixset=EPSG%3A6870&Service=WMTS&Request=GetTile&Version=1.0.0&Format=image%2Fjpeg&TileMatrix=EPSG%3A6870%3A5&TileCol={{X}}&TileRow={{Y}}"
```

**Comment by srdjantodorov93** [comment]
Thanks, but I have already tried with dezoomidy-rs, but unsuccessfully...

**Comment by lovasoa** [comment]
![map](https://user-images.githubusercontent.com/552629/79136956-22387f00-7db2-11ea-9474-36b3001b8d4b.jpg)

I had to greatly reduce the image quality to make it fit here, but the image you will download with dezoomify-rs will have the same quality as the original.

**Comment by lovasoa** [comment]
> Thanks, but I have already tried with dezoomidy-rs, but unsuccessfully...

What do you mean unsuccessfully ? I just gave you the url to pass to dezoomify-rs. I tested itt and it works.

**Comment by srdjantodorov93** [comment]
> > Thanks, but I have already tried with dezoomidy-rs, but unsuccessfully...
> 
> What do you mean unsuccessfully ? I just gave you the url to pass to dezoomify-rs. I tested it and it works.

Please, open link again. After 5-10 seconds of loading you will see that this map (satelite) is back-ordered, and topo map is on front.

**Comment by lovasoa** [comment]
Yes, this one works to, the same way. You take a tile URL, replace the x coordinate with `{{X}}` and they coordinate with `{{Y}}`, and dezoomify works with it. If you need more information about how to form the url templates, read the documentation of dezoomify and dezoomify-rs.

**Comment by lovasoa** [comment]
```
https://geoportal.asig.gov.al/service/wmts?layer=base_imagery%3A25k_krgjsh&style=default&tilematrixset=EPSG%3A6870&Service=WMTS&Request=GetTile&Version=1.0.0&Format=image%2Fjpeg&TileMatrix=EPSG%3A6870%3A5&TileCol={{X}}&TileRow={{Y}}'
```



![canvas](https://user-images.githubusercontent.com/552629/79139764-2dda7480-7db7-11ea-93d5-543358fe12d7.jpg)

**Comment by lovasoa** [comment]
I added basic support for wmts to https://dezoomify.ophir.dev

**Comment by lovasoa** [comment]
Bot-assisted triage note: closing this issue because it is currently labeled `solved`.
